<launch>
	<group ns="zone3">
		<group ns="chassis">
			<node name="can_rx_tli65" pkg="can_rx_tli65" type="can_rx_tli65">
				<!-- sub -->
				<param name="node_sub_topic" value="/zone3/drivers/can/rx" />
				<!-- pub -->
				<param name="container_topic" value="/zone3/chassis/rx/container" />
				<param name="EPB_topic" value="/zone3/chassis/rx/EPB" />
				<param name="Ft_topic" value="/zone3/chassis/rx/fault" />
				<param name="hands_topic" value="/zone3/chassis/rx/hands" />
				<param name="maintain_topic" value="/zone3/chassis/rx/maintain" />
				<param name="speed_topic" value="/zone3/chassis/rx/speed" />
				<param name="st_highspeed_topic" value="/zone3/chassis/rx/state_highspeed" />
				<param name="st_lowspeed_topic" value="/zone3/chassis/rx/state_lowspeed" />
				<param name="tyre_topic" value="/zone3/chassis/rx/tyre" />
				<param name="VIN_topic" value="/zone3/chassis/rx/VIN" />
				<param name="fuel_topic" value="/zone3/chassis/rx/fuel" />
			</node>
		</group>
	</group>
</launch>